ts/session/sending/MessageSender.ts

```diff
@@ -287,14 +289,25 @@ export async function sendToOpenGroupV2(
     filesToLink,
   });
 
-  const msg = await sendSogsMessageOnionV4(
-    roomInfos.serverUrl,
-    roomInfos.roomId,
-    new AbortController().signal,
-    v2Message,
-    blinded
-  );
-  return msg;
+  if (rawMessage.dataProto().reaction) {
+    const msg = await sendSogsReactionOnionV4(
+      roomInfos.serverUrl,
+      roomInfos.roomId,
+      new AbortController().signal,
+      rawMessage.dataProto().reaction as Reaction,
```

Comment:
rather than calling rawMessage.dataProto() here,
you could just make your reaction properti in OpenGroupVisibleMessage public so you can access it.

```
public readonly reaction?: Reaction;
``` in VisibleMessage.ts

Reply:
Nice, thank you.